src/Controller/Api/CollectionController.php

```diff
@@ -279,13 +280,13 @@ protected function getCollection($id)
      * @param Request     $request Symfony request
      * @param string|null $id      Collection identifier
      *
-     * @return Rest\View|FormInterface
+     * @return View|FormInterface
      */
     protected function handleWriteCollection($request, $id = null)
     {
         $collection = $id ? $this->getCollection($id) : null;
 
-        $form = $this->formFactory->createNamed('', 'sonata_classification_api_form_collection', $collection, [
```

Comment:
If this was working, should we provide a PR to psalm in order to not restrict the param to class-string ?

Reply:
Symfony changed the form types since 3.4 I think. They have to be class-string. I don't think this code was working/tested enough

Reply:
Also, tbh, I don't think anyone is using this api, they was so much wrong with it (previously we had null on the first parameter, another thing that it does not work)